src/components/Breakdown.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react'
+import Transactions from './Transactions'
+
+import '../styles/transactions.css'
+
+class Breakdown extends Component {
+    constructor() {
+        super()
+        this.state = {
+            show: {}
+        }
+    }
+
+    toggle = (c) => {
+        let tempShow = {...this.state.show}
+        tempShow[c] = !this.state.show[c]
+        this.setState({ show: tempShow })
+}
+    componentDidMount = async () => {
+        await this.props.updateTransactionsFromDB()
+        let tempShow = {}
+        this.props.categories && this.props.categories.map(c=> tempShow[c] = false)
+        this.setState({ show: tempShow })
+    }
+
+    render() {
+        const transactions = this.props.transactions
```

Comment:
deconstruct things. It is modern and cleaner

const {transactions, categories} = this.props